### PR TITLE
Updated timezone package, README.md

### PR DESCRIPTION
**Pull request by charles-cowart**
Updated README.md based on information in Google CI file.
Recent deprecations in timezone functionality required small changes.

### CODE REVIEW

**Comment by charles-cowart** [comment]
@wasade I can't request a reviewer. These changes may be useful.

File: README.md [hunk 1]
```diff
@@ -4,45 +4,38 @@
...
-```bash
-conda install redis
-pip install celery[redis]
-```
+`pip install -r ci/pip_requirements.txt`
 
 Then install the microsetta-private-api in editable mode:
 
-`pip install -e .`
+`pip install -e . --no-deps`	NOTE: NOT SURE ABOUT NO_DEPS FLAG
```
Comment (wasade) [inline README.md:27]: it should be correct, we want all deps to come from the ci/ files

File: microsetta_private_api/api/tests/test_api.py [hunk 2]
```diff
@@ -1953,11 +1954,12 @@ def test_get_sample_vioscreen_percent_energy_200(self):
...
         response_obj = json.loads(get_response.data)
         self.assertEqual(response_obj['sessionId'],
-                         vioscreen_percent_energy.sessionId)
+                         vioscreen_percent_e.sessionId)
         self.assertEqual(response_obj['calculations'][0]['code'],
-                         vioscreen_percent_energy.energy_components[0].code)
-        self.assertEqual(response_obj['calculations'][0]['amount'],
-                         vioscreen_percent_energy.energy_components[0].amount)
+                         vioscreen_percent_e.energy_components[0].code)
+        self.assertTrue(isclose(response_obj['calculations'][0]['amount'],
```
Comment (wasade) [inline microsetta_private_api/api/tests/test_api.py:1960]: good catch, thanks

**Review by wasade — APPROVED** [review]
minor questions but am +1

File: microsetta_private_api/admin/tests/test_admin_repo.py [hunk 3]
```diff
@@ -304,9 +303,8 @@ class AdminRepoTests(AdminTests):
     #  these fixed ones
     def test_retrieve_diagnostics_by_barcode_w_extra_info(self):
         def make_tz_datetime(y, m, d):
-            return datetime.datetime(y, m, d, 0, 0,
-                                     tzinfo=psycopg2.tz.FixedOffsetTimezone(
-                                         offset=-420, name=None))
+            return datetime(y, m, d, 0, 0,
```
[the remark this answers is not in the document]
Comment (charles-cowart) [reply microsetta_private_api/admin/tests/test_admin_repo.py:306]: When I first installed the software and dependencies, I got the following error when running the tests:
"AttributeError: module 'psycopg2' has no attribute 'tz'"
My guess was that the functionality had been removed from the current version of psycopg2, so I replaced it with similar functionality. (It looks like I have 2.9.2 installed)

File: README.md (hunk 1 above)
Comment (charles-cowart) [reply README.md:27]: Okay, I can add it back in.

File: microsetta_private_api/admin/tests/test_admin_repo.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (wasade) [reply microsetta_private_api/admin/tests/test_admin_repo.py:306]: oh interesting, and good catch. im 2.8.6 local

**Pull request merged**